contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -1115,13 +1116,12 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
    * @notice updates the signers and transmitters lists
    */
   function _updateTransmitters(address[] memory signers, address[] memory transmitters) internal {
+    uint256 transmittersListLength = s_transmittersList.length;
```

Comment:
nitpick - could we make this a `uint96` so that we don't have to cast it every time in the loop below? I think it will save a touch on gas